src/config.rs

```diff
@@ -58,12 +58,15 @@ impl SharedConfig {
         Ok(())
     }
 
-    pub fn get_icon(&self, icon: &str) -> Option<String> {
-        Some(
-            self.icons_format
-                .clone()
-                .replace("{icon}", self.icons.0.get(icon)?),
-        )
+    pub fn get_icon(&self, icon: &str) -> crate::errors::Result<String> {
+        use crate::errors::OptionExt;
+        Ok(self.icons_format.clone().replace(
+            "{icon}",
+            self.icons
+                .0
+                .get(icon)
+                .internal_error("get_icon()", &format!("icon '{}' not found", icon))?,
```

Comment:
Perhaps we could improve the error message here: tell the user to check their current theme's config is correct or something like that.